mods/ts/rules/gdi-infantry.yaml

```diff
@@ -47,6 +47,7 @@ MEDIC:
 		Weapon: Heal
 		TargetStances: Ally
 		ForceTargetStances: None
+		Cursor: heal
 	AttackFrontal:
```

Comment:
I'd set the `OutsideRangeCursor` to `heal` as well. The standard cursor looks out of place there.